cli/reference/blocks/stack.md

```diff
@@ -4,3 +4,70 @@ description: Learn how to configure stacks using the stack block.
 ---
 
 # The `stack` block
+
+Use the `stack` block to define a stack within a stack directory. Terramate detects stacks based on this `stack` block, and you can define only one `stack` block per stack directory. Use the `stack` block to configure:
+- The metadata of stack
+- The orchestration behavior of a stack
+- The change detection behavior of a stack
+
+### Metadata
+
+The stack metadata defined in the `stack` block is accessible only from the stack directory where it is defined. You can access this metadata via the `terramate` namespace using the `terramate.stack` object. For example, `terramate.stack.<key>` where `<key>` is any key defined in the `stack` block.
```

Comment:
The `accessible` could maybe use some leading context. In the context of terramate run? For globals? Unless this is part of a bigger picture and the context is obvious, but to me looking at this part only, it's not.

Reply:
I replaced 'accessible' with 'scoped locally'. Will that be good?